src/store/listAnime.js

```diff
@@ -301,4 +301,10 @@ export default [
         animeYear:2013,
         animeLink:"https://myanimelist.net/anime/18179/Yowamushi_Pedal"
     }
+	{
+        animeEmoji:"🤵🏼�🤟🏼",
+        animeName:"Koe_no_Katachi",
+        animeYear:2016,
+        animeLink:"https://myanimelist.net/anime/28851/Koe_no_Katachi"
+    },
```

Comment:
this comma cause error , and why the title noragami but the anime name is koe no katachi?